WordPressCom-Stats-iOS/InsightsTableViewController.m

```diff
@@ -937,6 +1045,13 @@ - (void)configureTwoColumnRowCell:(UITableViewCell *)cell
 }
 
 
+- (void)configureInsightsWrappingTextCell:(StatsStandardBorderedTableViewCell *)cell
+{
+    UILabel *label = (UILabel *)[cell.contentView viewWithTag:100];
```

Comment:
(And this is me, not saying anything about outlets :stuck_out_tongue_winking_eye: )